data/xml/2021.lchange.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.lchange">
+  <volume id="1" ingest-date="2021-07-20">
+    <meta>
+      <booktitle>Proceedings of The 2nd International Workshop on Computational Approaches to Historical Language Change 2021</booktitle>
+      <editor><first>workshop chair</first><last>Nina Tahmasebi</last></editor>
```

Comment:
"workshop chair" in name field

Reply:
@danielgildea thank you for pointing this out!